src/components/contact.tsx

```diff
@@ -172,8 +175,7 @@ export default function Signup(props: { headline: boolean }) {
 
       <div className="control">
         <label htmlFor="textBox">מידע נוסף</label>
-        <textarea
-        className="speaker-card-text-default"
+        <textarea className="control-text-input"
```

Comment:
Once you have mui in context, you no longer need to work with semantic HTML. Please look at the mui API and find the correlated component. You will get the styling "free."

This comment refers to this PR in general and the project.